.github/workflows/benchmark.yml

```diff
@@ -5,11 +5,12 @@ on:
     branches:
       - "master"
   pull_request:
-    paths: 
-       - forc-plugins/forc-lsp/** 
-       - sway-lsp/** 
-       - forc-pkg/** 
-       - sway-core/**
+#    paths:
+#       - forc-plugins/forc-lsp/**
+#       - sway-lsp/**
+#       - forc-pkg/**
+#       - sway-core/**
```

Comment:
we probably want to add these conditions back in as these are the only workspace members that have an impact on the the LSP benchmarks that are run by codspeed

Reply:
haha just as I was typing this :)

Reply:
This was just temporary so I could see the job actually run and verify it all worked correctly on buildjet